incubator/kubeless/crds/cronjobtrigger-crd.yaml

```diff
@@ -1,10 +1,12 @@
 apiVersion: apiextensions.k8s.io/v1beta1
-description: CRD object for Cron trigger type
 kind: CustomResourceDefinition
 metadata:
   name: cronjobtriggers.kubeless.io
   labels:
-{{ include "labels.standard" . | indent 4 }}
+    app: kubeless
+  annotations:
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
```

Comment:
why are you adding the delete-policy? Also, I think that the `crd-install` annotation is no longer needed.

Reply:
You're right, but this combined with the crds.yaml file is a temporary hack so that the chart can support both helm 2 and 3.

See #19008, I'm guessing kubeless was missed because it was in the incubator repo rather than stable.

Alternatively we could require a minimum helm version of 3 for the chart, but I think this repo aims to support both until EOL for helm 2.

Reply:
:+1: got it